src/test/java/org/springframework/data/couchbase/util/ClusterAwareIntegrationTests.java

```diff
@@ -62,13 +64,33 @@ public static String bucketName() {
 	 * @return the connection string to connect.
 	 */
 	public static String connectionString() {
+		/*
 		return seedNodes().stream().map(s -> {
 			if (s.kvPort().isPresent()) {
-				return s.address() + ":" + s.kvPort().get();
+				return s.address() + ":" + s.kvPort().get() + "=" + Services.KV;
+			} else if (s.clusterManagerPort().isPresent()) {
+				return s.address() + ":" + s.clusterManagerPort().get() + "=" + Services.MANAGER;
 			} else {
-				return s.address();
+				return s.address() ;
 			}
 		}).collect(Collectors.joining(","));
+		*/
+		StringBuffer sb = new StringBuffer();
```

Comment:
How are the changes in this file related?

Reply:
The previous code did not supply the management port.  If a config-refresh is attempted while the tests are running, the queries time-out.